pywbem/_subscription_manager.py

```diff
@@ -407,14 +399,24 @@ def add_server(self, server):
                 self._owned_destinations[server_id].append(inst)
 
         filter_name_pattern = re.compile(
+            _format(r'^pywbemfilter:{0}:[^:]*$',
+                    self._subscription_manager_id))
+        filter_name_old_pattern = re.compile(  # before pywbem 1.3
             _format(r'^pywbemfilter:owned:{0}:{1}:[^:]*:[^:]*$',
                     this_client, self._subscription_manager_id))
+
         filter_insts = server.conn.EnumerateInstances(
             FILTER_CLASSNAME, namespace=server.interop_ns)
 
         for inst in filter_insts:
             if re.match(filter_name_pattern, inst.path.keybindings['Name']):
                 self._owned_filters[server_id].append(inst)
+            if re.match(filter_name_old_pattern, inst.path.keybindings['Name']):
+                warnings.warn(
+                    _format(
+                        "Ignored owned indication filter with old name format "
+                        "in WBEM server: {0}", inst.path),
```

Comment:
Add "received from WBEM server" to text. Since this warning sort of appears out of the clear blue (i.e starting the manager)

Reply:
Changed to: "Ignored owned indication filter instance in WBEM server that has the old name format: {0}"